### PR TITLE
fix: plus minus icon size

### PR DESCRIPTION
**Pull request by a-m-dev**
Plus/minus icon shrinking in the size when text requires bigger space. This PR fixes it.

Before: 
Screenshot 2023-10-23 at 14 50 43

After:
Screenshot 2023-10-23 at 14 50 06

### CODE REVIEW

File: src/pivot-table/components/icons/styles.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { HEADER_ICON_SIZE } from "../../constants";
+import { baseFlex } from "../shared-styles";
+
+export const headerPlusMinusIconWrapperStyle = {
+  width: HEADER_ICON_SIZE,
```
Comment (cbt1) [inline src/pivot-table/components/icons/styles.ts:5]: `HEADER_ICON_SIZE` is 12px but the icons are 13px. That is likely why the rendering tests are failing with "fuzzy" icons

Comment (a-m-dev) [reply src/pivot-table/components/icons/styles.ts:5]: will update it soon

File: src/pivot-table/components/icons/styles.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { baseFlex } from "../shared-styles";
+
+export const headerPlusMinusIconWrapperStyle = {
+  width: 13,
```
Comment (cbt1) [inline src/pivot-table/components/icons/styles.ts:4]: Maybe expose this pixel value as a constant and also consume it in the icon components?

**Review by cbt1 — APPROVED** [review]
Nice. Any idea as to why the icons scaled the way it did?

**Comment by a-m-dev** [comment]
I think because `HEADER_ICON_SIZE` was 12 px and the width of icon we pass to it is 13px. 
so the svg icon will try to fit itself into the 12px wrapper div -> which will result in shrieked size

**Comment by cbt1** [comment]
> I think because `HEADER_ICON_SIZE` was 12 px and the width of icon we pass to it is 13px. so the svg icon will try to fit itself into the 12px wrapper div -> which will result in shrieked size

I mean the "fix" part of this PR. Not the comment I had earlier.

**Review by cbt1 — APPROVED** [review]

File: src/pivot-table/components/shared-styles.ts [hunk 3]
```diff
@@ -14,12 +14,16 @@ export enum Colors {
...
 export const CELL_PADDING = 4;
 
-export const topContainerCellStyle: React.CSSProperties = {
+export const baseFlex: React.CSSProperties = {
   display: "flex",
   justifyContent: "center",
   alignItems: "center",
 };
 
+export const topContainerCellStyle: React.CSSProperties = {
```
[the remark this answers is not in the document]
Comment (cbt1) [reply src/pivot-table/components/shared-styles.ts:23]: I will remove that in my refactor dimension cells PR. As that styling is no longer needed at all

**Pull request merged**